### DISCUSSION TITLE
Dyson Sphere + Dyson Swarm Synchronization

### DISCUSSION THREAD

**Discussion by Baldie-dev** [NebulaModTeam/nebula#66] [General]
Hi all,
Since  I was going multiple times through the code that loads / creates / calculates and renders each component of the Dyson sphere and it was a major showstopper for the dedicated server I can start working on the synchronization if nobody is currently doing it.
I would like to help, but I am not sure with what.

Thanks!

**Comment by Mushroom** [comment]
If you don't see it as an issue, create it, and if you cannot assign yourself one of us will assign you :)

If you want to work on the Dyson Sphere syncing, I don't think anybody is doing that at the moment!

Alternatively if you want to get a little more acquainted with the project, have a look through the open and unassigned issues, and pick one you like the look of (and hasn't been worked on/deprioritised by someone else).
